hypothesis-python/tests/numpy/test_gen_data.py

```diff
@@ -1194,7 +1202,11 @@ def test_basic_indices_can_generate_long_ellipsis():
     )
 
 
-@given(nps.basic_indices(shape=(0, 0, 0, 0, 0)).filter(lambda idx: Ellipsis in idx))
+@given(
+    nps.basic_indices(shape=(0, 0, 0, 0, 0)).filter(
+        lambda idx: isinstance(idx, Iterable) and Ellipsis in idx
```

Comment:
```suggestion
        lambda idx: isinstance(idx, tuple) and Ellipsis in idx
```

No need to be fancy here, it's either a `tuple` or a single element.

Reply:
That's true.